sys/src/9/amd64/mmu.c

```diff
@@ -700,7 +700,7 @@ vunmap(void* v, usize size)
 	DBG("vunmap(%#p, %lu)\n", v, size);
 
 	if(machp()->machno != 0)
-		print("vmap: machp()->machno != 0\n");
+		DBG("vmap: machp()->machno != 0\n");
```

Comment:
Why does this go from print -> DBG, when the change above goes from DBG -> print?  Should they be consistent?

Reply:
Both above have to be in the same way, yes. This is a consecuence of try and error. DBG doesn't avoid the print if I define it as print. DBFLG is used in mmu and if present it doesn't print. There were a DBG for the first vmap doing nothing because DBG wasn't defined. When I defined DBG print, a thousand of lines rised during boot process. That's why I left it as it was and disabled machno!=0 with the flag. I tried to be less intrusive as possible. I think the right way is using if DBFLG as in the rest of file. Without macro.

Reply:
I, I wrote from my phone. Yes I used DBG in parallel with previous DBG for vunmap